lib/easybill-rest-client/generic_api.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+module EasybillRestClient
+  module GenericApi
+    def self.new(resource_name, resource_class)
```

Comment:
I was wondering the same. It saves the class methods in a few classes, though. On the other hand, I stumbled over this when I tried to write a test already, since I assumed when I call `new`, what I get is an instance of the class I call it on, but I ended up with said awkward anonymous class. We could at least rename the method. Or simply keep it simple. :wink: